test/system/scenarios/invalid-orders.js

```diff
@@ -0,0 +1,84 @@
+const Promish = require('promish');
+
+const run = (trader1, trader2, trader3) => {
+  console.log('****************************************************************');
+  console.log('** Scenario 13: Test Invalid Orders                         **');
+  console.log('****************************************************************');
+
+  return Promish.resolve()
+    .then(() => testMarketOrder(trader1))
+    .then(() => testLimitOrder(trader2))
+    .then(() => testStopOrder(trader3))
+    .then(() => testStopLimitOrder(trader2))
+    .then(() => testTrailingStopOrder(trader3));
+};
+
+function testMarketOrder(trader1) {
+  return Promish.resolve()
+    .then(() => trader1.createOrder({ market: 'BTC', type: 'M', side: 'B', quantity: 1}, { expectFail: true , expectFailReason: 'due to invalid market'}))
+    .then(() => trader1.createOrder({ type: 'M', side: 'S' }, { expectFail: true, expectFailReason: 'market sell order without qty'}))
+    .then(() => trader1.createOrder({ type: 'M', side: 'B', quantity: 1 }, { expectFail: true, expectFailReason: 'market buy order with qty instead value' }))
+    .then(() => trader1.createOrder({ type: 'M', quantity: 1 }, { expectFail: true, expectFailReason: 'market order without side'}))
+    .then(() => trader1.createOrder({ type: 'M', side: 'S', quantity: 0 }, { expectFail: true, expectFailReason: 'market order with 0 qty' }))
+    .then(() => trader1.createOrder({ type: 'M', side: 'S', quantity: -0.5 }, { expectFail: true, expectFailReason: 'market order with negative qty' }))
+    .then(() => trader1.createOrder({ type: 'M', side: 'S', quantity: 'd' }, { expectFail: true, expectFailReason: 'market order with qty in string' }))
+    .then(() => trader1.createOrder({ type: 'M', side: 1, quantity: 0.5 }, { expectFail: true, expectFailReason: 'market order with side as integer' }))
+    .then(() => trader1.createOrder({ type: 'M', side: 'B', value: 0 }, { expectFail: true, expectFailReason: 'market buy order with 0 value' }))
+    .then(() => trader1.createOrder({ side: 'S', quantity: 1 }, { expectFail: true, expectFailReason: 'market buy order with 0 value' }));
+}
+
+function testLimitOrder(trader2) {
```

Comment:
Also test for
* invalid quantity (zero, negative, string)
* invalid price (negative, string)